src/test/scala/ch/epfl/bluebrain/nexus/kg/TestHelper.scala

```diff
@@ -40,4 +43,26 @@ trait TestHelper {
 
   def uuid = UUID.randomUUID().toString
 
+  implicit val jsonEq: Equality[Json] = new Equality[Json] {
```

Comment:
this is dangerous as order of elements in an array it's important; it may make sense for json-ld if the container type is a `@set`

I would recommend implementing a word which allows choosing when the order doesn't matter